### PR TITLE
Add permission to update namespace finalizer

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

## Proposed Changes

- like https://github.com/knative/eventing/pull/5501 
-
-

**Release Note**

```release-note
source and channel webhook cluster role has update permission for namespaces/finalizers.

```

**Docs**

### CODE REVIEW

File: config/source/single/roles/clusterrole.yaml [hunk 1]
```diff
@@ -98,6 +98,14 @@ rules:
   - watch
   - patch
 
+  # finalizers are needed for the owner reference of the webhook
+  - apiGroups:
+      - ""
+    resources:
+      - "namespaces/finalizers"
+    verbs:
+      - "update"
```
Comment (pierDipi) [inline config/source/single/roles/clusterrole.yaml:107]: This should be indented one level less.

Comment (matzew) [reply config/source/single/roles/clusterrole.yaml:107]: :facepalm:

Comment (matzew) [reply config/source/single/roles/clusterrole.yaml]: @pierDipi updated

**Comment by matzew** [comment]
/unhold

the verify now works.... w/o no change ... (I just rebased)

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Pull request merged**